src/style.css

```diff
@@ -0,0 +1,33 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--0.stars__star {
```

Comment:
The selector '.stars--0.stars__star' is not needed as the default stars styling is already applied in '.stars__star'